### DISCUSSION TITLE
New Placeholder

### DISCUSSION THREAD

**Discussion by DarkSavci** [BG-Software-LLC/SuperiorSkyblock2#1662] [Pending Feature Requests] [answered]
Global Broken Block Count placeholder would be nice.

**Comment by Lemvs** [comment, marked as answer]
This data is not stored anywhere by SuperiorSkyblock, you can use Minecraft statistics and Statistics Expansion for PlaceholderAPI for it.

@OmerBenGera you can mark this answered and close it.
